tests/server/test_base_server.py

```diff
@@ -14,7 +14,7 @@
 model = LinearTransform()
 
 
-class TestFLServer(FlServer):
+class DummyFLServer(FlServer):
```

Comment:
This was generating a warning in our test collection because it was fronted by the word test. Just changing it to suppress the error.